### PR TITLE
gdb: add version 7.11

### PR DESCRIPTION
**Pull request by sknsean**
I know it's one version old, but i think we should wait for some yocto patches before upgrading

### CODE REVIEW

File: recipes/gdb/files/0005-Add-support-for-Renesas-SH-sh4-architecture.patch [hunk 1]
```diff
@@ -0,0 +1,921 @@
+From 8c5fe58c5a0044ddb517a41b277ed27fb3d3bedc Mon Sep 17 00:00:00 2001
+From: Khem Raj <raj.khem@gmail.com>
+Date: Mon, 2 Mar 2015 02:31:12 +0000
+Subject: [PATCH 05/10] Add support for Renesas SH (sh4) architecture.
```
Comment (kimrhh) [inline recipes/gdb/files/0005-Add-support-for-Renesas-SH-sh4-architecture.patch]: uhm. why?

File: recipes/gdb/files/0006-Dont-disable-libreadline.a-when-using-disable-static.patch [hunk 2]
```diff
@@ -0,0 +1,50 @@
+From f3932cb2960fd54655a448b13d5a5b80f356f8de Mon Sep 17 00:00:00 2001
+From: Khem Raj <raj.khem@gmail.com>
+Date: Sat, 30 Apr 2016 15:25:03 -0700
+Subject: [PATCH 06/10] Dont disable libreadline.a when using --disable-static
```
Comment (kimrhh) [inline recipes/gdb/files/0006-Dont-disable-libreadline.a-when-using-disable-static.patch]: is this a problem we actually have or did you just steal all the patches? :)

File: recipes/gdb/files/0007-use-asm-sgidefs.h.patch [hunk 3]
```diff
@@ -0,0 +1,36 @@
+From 048675a915a72989f2613386975730da016e7c5d Mon Sep 17 00:00:00 2001
+From: Andre McCurdy <amccurdy@gmail.com>
+Date: Sat, 30 Apr 2016 15:29:06 -0700
+Subject: [PATCH 07/10] use <asm/sgidefs.h>
+
+Build fix for MIPS with musl libc
```
Comment (kimrhh) [inline recipes/gdb/files/0007-use-asm-sgidefs.h.patch]: we have no MIPS, let us please keep patches to a minimum

File: recipes/gdb/files/0006-Dont-disable-libreadline.a-when-using-disable-static.patch (hunk 2 above)
Comment (sknsean) [inline recipes/gdb/files/0006-Dont-disable-libreadline.a-when-using-disable-static.patch]: yes from openembedded

**Comment by kimrhh** [comment]
only thing left is to not add the mips patch

**Pull request merged**